plugins/arDominionB5Plugin/modules/settings/templates/treeviewSuccess.php

```diff
@@ -82,40 +76,36 @@
             <div class="row">
 
               <div class="col-md-4">
-                <?php echo $form->showIdentifier
-                    ->label(__('Show identifier'))
-                    ->renderRow(); ?>
+                <?php echo render_field($form->showIdentifier
+                    ->label(__('Show identifier'))); ?>
               </div>
 
               <div class="col-md-4">
-                <?php echo $form->showLevelOfDescription
-                    ->label(__('Show level of description'))
-                    ->renderRow(); ?>
+                <?php echo render_field($form->showLevelOfDescription
+                    ->label(__('Show level of description'))); ?>
               </div>
 
               <div class="col-md-4">
-                <?php echo $form->showDates
-                    ->label(__('Show dates'))
-                    ->renderRow(); ?>
+                <?php echo render_field($form->showDates
+                    ->label(__('Show dates'))); ?>
               </div>
 
             </div>
 
-            <p>
-              <?php echo $form->fullItemsPerPage
-                  ->label(__('Items per page'))
-                  ->help(
-                    __('Items per page can be a minimum of %1% and a maximum of %2%',
-                      [
-                          '%1%' => format_number(10),
-                          '%2%' => format_number(
-                            sfConfig::get('app_treeview_items_per_page_max', 10000)
-                          ),
-                      ]
-                    )
-                  )
-                  ->renderRow(); ?>
-            </p>
+            <?php echo render_field(
```

Comment:
Should this be wrapped in a `div.col-md-4` and put in the same container of rows as the other fields above?

Reply:
Thanks, will do. I've used col-md-12 to dedicate a full line, matching the old layout.